experiment/simulation/index.html

```diff
@@ -1,13 +1,275 @@
+          fill="url(#half_grad10)"
+          fill="#00ccff"
+          opacity="1.00"
+          d=" M 405.85 176.62 C 409.22 176.62 412.59 176.62 415.96 176.63 C 416.05 200.42 415.98 224.20 415.99 247.98 C 412.66 247.97 409.33 248.02 406.01 248.17 C 405.58 224.33 405.81 200.47 405.85 176.62 Z"
+        />
+        <path
+          id="sixth-tube"
+          fill="url(#half_grad11)"
+          fill="#00ccff"
+          opacity="1.00"
+          d=" M 440.05 191.04 C 443.92 191.00 447.80 191.00 451.68 190.98 C 451.85 209.97 451.70 228.96 451.72 247.94 C 447.81 248.12 443.91 248.11 440.01 247.97 C 440.04 228.99 439.95 210.01 440.05 191.04 Z"
+        />
+        <path
+          id="seventh-tube"
+          fill="url(#half_grad12)"
+          fill="#00ccff"
+          opacity="1.00"
+          d=" M 471.34 243.16 C 471.62 230.67 470.49 218.09 472.08 205.67 C 475.63 206.53 479.29 206.53 482.92 206.38 C 483.07 220.25 482.95 234.12 482.97 247.99 C 479.13 247.96 475.30 248.01 471.47 248.17 C 471.41 246.50 471.36 244.83 471.34 243.16 Z"
+        />
+        <path
+          id="second-container"
+          fill="url(#half_grad13)"
+          fill="#00ccff"
+          opacity="1.00"
+          d=" M 501.02 131.24 C 553.49 131.27 605.97 131.23 658.44 131.27 C 658.50 156.51 658.46 181.75 658.47 206.99 C 658.52 218.26 658.17 229.54 658.95 240.79 C 669.26 241.10 679.58 240.96 689.90 241.03 C 689.83 242.84 689.78 244.65 689.73 246.47 C 679.44 246.75 669.14 246.46 658.86 246.87 C 658.02 265.63 658.70 284.41 658.45 303.19 C 605.98 303.21 553.51 303.20 501.04 303.19 C 500.99 245.87 500.99 188.56 501.02 131.24 Z"
+        />
+        <path
+          id="tap2-pipe-after"
+          fill="url(#half_grad14)"
+          fill="#00ccff"
+          opacity="1.00"
+          d=" M 713.54 241.17 C 725.94 240.81 738.35 241.05 750.75 241.05 C 750.75 243.02 750.76 244.99 750.79 246.96 C 747.18 246.99 743.57 247.10 739.98 246.83 C 731.24 246.22 722.46 247.03 713.73 246.35 C 713.69 244.62 713.62 242.89 713.54 241.17 Z"
+        />
+        <path
+          id="tap2-stream-before"
+          fill="url(#half_grad15)"
+          fill="#00ccff"
+          opacity="1.00"
+          d=" M 741.51 241.01 C 744.59 240.99 747.68 240.99 750.77 241.07 C 750.82 253.73 750.47 266.39 750.98 279.05 C 751.02 316.34 751.00 353.62 750.99 390.91 C 747.84 390.90 744.69 390.90 741.56 390.94 C 741.47 360.96 741.54 330.98 741.52 301.01 C 741.64 292.33 741.17 283.64 741.95 274.99 C 742.16 263.65 742.01 252.32 741.51 241.01 Z"
+        />
+        <path
+          id="last-container"
+          fill="url(#half_grad16)"
+          fill="#00ccff"
+          opacity="1.00"
+          d=" M 692.99 334.52 C 756.84 334.53 820.68 334.51 884.52 334.53 C 884.41 353.34 884.62 372.16 884.43 390.98 C 820.62 390.85 756.81 390.87 693.01 390.98 C 692.94 372.16 693.01 353.34 692.99 334.52 Z"
+        />
+      </g>
+    </svg>
+    <!-- Js files -->
+    <script src="./js/main.js"></script>
```

Comment:
Would you consider including main.js after the other scripts, which main,js may be depending on?

Reply:
For this experiment in particular, main.js is not dependent on any other script. Though if still needed we can include it at the end